plugins/package-managers/node/src/main/kotlin/npm/Npm.kt

```diff
@@ -84,9 +84,9 @@ class Npm(override val descriptor: PluginDescriptor = NpmFactory.descriptor, pri
 
     private lateinit var stash: DirectoryStash
 
-    private val moduleInfoResolver = ModuleInfoResolver.create { _, moduleId: String ->
+    private val moduleInfoResolver = ModuleInfoResolver.create { workingDir: File, moduleId: String ->
```

Comment:
Commit message nits:
- I believe it's fine to link to the same issue only once, so I'd drop "see [1]".
- We usually say "Fixes #10272" without a colon.

Reply:
Same comments about omitting types applies for this commit.